deploy/cephfs/kubernetes/cephfs-storage-class.yaml

```diff
@@ -4,5 +4,9 @@ metadata:
   name: cephfs
 provisioner: cephfsplugin
 parameters:
-  provisionRoot: /cephfs
-reclaimPolicy: Delete 
+  adminID: admin
```

Comment:
With CSI Spec 0.2.0 you do not have to keep ceph username/key in StorageClass. Please refer to updated CSI RBD driver StorageClass and also controller.go to see how to use credentails which are passed now inside of RPC call.